src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -35,6 +35,7 @@ public class WebxdcActivity extends WebViewActivity implements DcEventCenter.DcE
   private static final String TAG = WebxdcActivity.class.getSimpleName();
   private DcContext dcContext;
   private DcMsg dcAppMsg;
+  private String baseURL;
```

Comment:
is that set somewhere?

Reply:
`*sigh*` after implementing all in this file and testing in deltalab I had to revert the file to be able to checkout upstream/master and for got to re-add this, will fix now

Reply:
fixed